shared/controller-api/src/main/java/io/pravega/shared/controller/event/kvtable/CreateTableEvent.java

```diff
@@ -83,6 +84,7 @@ private void write00(CreateTableEvent e, RevisionDataOutput target) throws IOExc
             target.writeUUID(e.tableId);
             target.writeInt(e.primaryKeyLength);
             target.writeInt(e.secondaryKeyLength);
+            target.writeLong(e.rolloverSizeBytes);
```

Comment:
This isn't backwards compatible. You need to create Revision 1 where you (only) write the rollover size (and of course, where you read it from).

You'll need to create `write01` and `read01` and then in `declareVersions` just tag another `.revision(1, ...)` to that statement.